backend/src/main/resources/application.properties

```
@@ -1 +1,7 @@
-spring.data.mongodb.uri=mongodb://ztbzBackend:example@localhost:27017/ztbzDatabase
\ No newline at end of file
+spring.data.mongodb.uri=mongodb://ztbzBackend:example@localhost:27017/ztbzDatabase
+spring.datasource.url=jdbc:postgresql://postgres:5432/default_database
```

I think we should set here `localhost` for running application directly from IDE and set proper ENV in docker-compose file 
```suggestion
spring.datasource.url=jdbc:postgresql://localhost:5432/default_database
```

---

maybe yes but we communicate only in docker environment, we are launching the app as a whole